app/assets/javascripts/admin/util.js.erb

```diff
@@ -18,6 +18,7 @@ $(document).ready(function() {
     }
   }
   window.FLATPICKR_DATE_DEFAULT = {
+    mode: "range",
```

Comment:
Not all flatpickr instances will use this mode. We should add this options only where it's relevant.